### PR TITLE
Fixed one of the tests in cyclical learning rate.

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
There was an attribute error on learning_rate_schedule and an off by one error in the tests. The step size was also doubled.

I also reduced the step size to make the test much faster.

### CODE REVIEW

File: tensorflow_addons/optimizers/tests/cyclical_learning_rate_test.py [hunk 1]
```diff
@@ -25,42 +26,40 @@
...
-        self.evaluate(tf.compat.v1.global_variables_initializer())
-        expected = np.concatenate(
-            [
-                np.linspace(initial_learning_rate, maximal_learning_rate, num=2001)[1:],
-                np.linspace(maximal_learning_rate, initial_learning_rate, num=2001)[1:],
-            ]
-        )
+    expected = np.concatenate(
+        [
+            np.linspace(initial_learning_rate, max_learning_rate, num=step_size + 1),
```
Comment (AakashKumarNain) [inline tensorflow_addons/optimizers/tests/cyclical_learning_rate_test.py:49]: I didn't look at the full code yet but in the previous version this was `np.linspace(initial_learning_rate, maximal_learning_rate, num=num_steps+!)[1:]`. Is this right?

Comment (gabrieldemarmiesse) [reply tensorflow_addons/optimizers/tests/cyclical_learning_rate_test.py:49]: Yes

Comment (AakashKumarNain) [reply tensorflow_addons/optimizers/tests/cyclical_learning_rate_test.py:49]: Okay. Cool

**Comment by gabrieldemarmiesse** [comment]
Thank you for the review!

**Comment by AakashKumarNain** [comment]
No worries. Thanks for updating this
